### DISCUSSION TITLE
If the computer is shut down or the program is interrupted, can I continue to run from the last running time?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#804] [Q&A] [answered]
Does SeisSol support this function?

**Comment by krenzland** [comment, marked as answer]
It doesn't support it currently; however this feature is on our long-term TODO list.

**Reply by smallriver666** [reply]
> It doesn't support it currently; however this feature is on our long-term TODO list.

This is a very powerful function, looking forward to its realization in the future.
